lib/wash/exe/cat.js

```diff
@@ -25,41 +26,64 @@ var JsExecuteContext;
 export var main = function(cx) {
   cx.ready();
 
-  var list = cx.getArg('_', []);
-  if (!list.length || cx.getArg('help')) {
+  if (cx.getArg('help')) {
     cx.stdout.write([
-      'usage: cat <file> ...',
-      'Echo one or more files to stdout.'
+      'usage: cat [<file>|-]...',
+      '',
+      'Reads files sequentially, writing them to the standard output.',
+      'The file operands are processed in command-line order.',
+      '',
+      'If file is a single dash (`-`) or absent, cat reads from the standard',
+      'input.'
     ].join('\r\n') + '\r\n');
     cx.closeOk();
     return;
   }
 
-  var fileSystem = cx.fileSystemManager;
+  // If no command line arguments were provided, default to 'read from stdin'.
+  var list = cx.getArg('_', ['-']);
   var errors = false;
+  var stdinReader = new Completer();
+
+  if (list.indexOf('-') !== -1) {
+    var stdinInput = '';
+    cx.stdin.pause();
```

Comment:
This should be a no-op, as commands are guaranteed to receive `paused` readable streams (e.g. `stdin`), right?